gridt/src/app/about/about.page.scss

```diff
@@ -0,0 +1,25 @@
+ion-slide {
+    height: 100vh;
+    flex-direction: column;
+    text-align: justify;
+}
+
+ion-img {
+    max-width: 50vw;
+    max-height: 50vw;
+    overflow: hidden;
+}
+
+p {
+    padding: 1em;
+}
```

Comment:
It is a good idea to limit addressing tags directly, instead use something like `ion-slides.intro-slides`, or something along those lines to make sure you don't change classes in other parts of the application, as well as keeping your intentions very clear.